components/usage-api/usage/v1/usage.proto

```diff
@@ -70,6 +73,51 @@ message PaginatedResponse {
     int64 page = 5;
 }
 
+message FindUsageRequest {
+    string attribution_id = 1;
+
+    // from specifies the starting time range for this request.
+    // All instances which existed starting at from will be returned.
+    google.protobuf.Timestamp from = 2;
+
+    // to specifies the end time range for this request.
+    // All instances which existed ending at to will be returned.
+    google.protobuf.Timestamp to = 3;
+
+    enum Ordering {
+        ORDERING_DESCENDING = 0;
+        ORDERING_ASCENDING = 1;
+    }
+
+    Ordering order = 4;
+
+    PaginatedRequest pagination = 5;
+}
+
+message FindUsageResponse {
+    repeated BilledSession sessions = 1;
+    double total_credits_used = 2;
```

Comment:
If I understand this correctly then this would return the total credits for the period in the request, correct? For scenarios like this, the proto is a good place to put comments with this as they are included in all of the code-generated clients so docs can be right there with the SDK